src/coffee/run.coffee

```diff
@@ -280,7 +280,7 @@ module.exports = class
       .option '-o, --out <file>', 'Path to the file the exporter will write the resulting CSV in'
       .option '-j, --json', 'Export in JSON format'
       .option '-q, --queryString <query>', 'Query string to specify the sub-set of products to export'
-      .option '-l, --language [lang]', 'Language used on export for category names (default is en)', 'en'
+      .option 'Language used on export for localised attributes (except lenums) and category names (default is en)'
```

Comment:
not sure what the original thought was but it makes sense, why should it only work for category names?!
I tested it and it works for all localised attributes (except now of course labels of Lenum)